de.metas.salescandidate.base/src/main/java/de/metas/ordercandidate/api/OLCandCreateRequest.java

```diff
@@ -128,8 +144,8 @@ private OLCandCreateRequest(
 
 		this.orgId = orgId;
 
-		this.dataDestInternalName = assumeNotEmpty(dataDestInternalName, "dataDestInternalName may not be empty");
-		this.dataSourceInternalName = assumeNotEmpty(dataSourceInternalName, "dataSourceInternalName may not be empty");
+		this.dataSourceId = assumeNotNull(dataSourceId, "data source may not be empty");
```

Comment:
afaiu  assumeNotNull is not needed, since you have `@NonNull`

Reply:
Right. Fixed.